src/workflows-app/RunDetails.js

```diff
@@ -251,7 +255,24 @@ export const BaseRunDetails = (
           div({ style: { padding: '1rem 2rem 2rem' } }, [header]),
           div({ style: { display: 'flex', justifyContent: 'space-between', padding: '1rem 2rem 2rem' } }, [
             h(WorkflowInfoBox, { workflow }, []),
-            h(TroubleshootingBox, { name, namespace, logUri: workflow.workflowLog, submissionId, workflowId, showLogModal }, []),
+            h(
+              TroubleshootingBox,
+              {
+                name,
+                namespace,
+                logUri: workflow.workflowLog,
+                submissionId,
+                workflowId,
+                showLogModal,
+                appId: appIdMatched,
+                workflowName: appIdMatched
+                  ? stdOut
+                      .substring(appIdMatched.index + appIdMatched[0].length + 1)
```

Comment:
Small readability nit: I think it would be nice for future maintainers if this ternary was instead a named function (maybe even with a short comment explaining what it's for)